### ISSUE TITLE
Compatibility with OS X and Safari

### ISSUE THREAD

**Issue by SimonKis** [VodkaBears/Vide#146]
Hi, 
have you someone experience with compatibility on OS X in Safari browser? I need know if it works.

**Comment by nivcaner** [comment]
yes. it works

**Comment by phazonoverload** [comment]
Not for me. Was there any additional configuration needed?

**Comment by jimzarkadas** [comment]
@phazonoverload did you find any solution for this ? I have the same issue :-/

**Comment by walkerdigital** [comment]
In Safari I noticed that the entire video file apparently needs to load before it will start playing in browser.  I haven't noticed any issue on Chrome or FF whereas it plays immediately.  In Safari it's taking about 20 seconds on slower connection.

**Comment by zsitro** [comment]
@walkerdigital any workaround for this? Same issue here. Maybe video format problem?

**Comment by zsitro** [comment]
Yes check "Support MP4 Flat and Hint" settings here which solved for me.
http://www.any-video-converter.com/mac/online_help_free/preferences-setting.php
so this issue is not related to this plugin, should be closed!
